trollflow2/plugins/__init__.py

```diff
@@ -1114,3 +1114,37 @@ def callback_close(obj, targs, job, fmat_config):
         for targ in targs:
             targ.close()
     return obj
+
+
+def use_fsspec_cache(job):
+    """Use the caching from fsspec for (remote) files."""
+    import fsspec
+    from satpy.readers import FSFile
+
+    cache = job["product_list"]["fsspec_cache"]
+    cache_dir = job["product_list"].get("fsspec_cache_dir")
+    filenames = job["input_filenames"]
+    cached_filenames = [f"{cache}::{f}" for f in filenames]
+    if cache_dir:
+        if cache == "blockcache":
+            open_files = fsspec.open_files(cached_filenames,
+                                           blockcache={"cache_storage": cache_dir})
+        elif cache == "filecache":
+            open_files = fsspec.open_files(cached_filenames,
+                                           filecache={"cache_storage": cache_dir})
+        elif cache == "simplecache":
+            open_files = fsspec.open_files(cached_filenames,
+                                           simplecache={"cache_storage": cache_dir})
```

Comment:
how about
```python
kwargs = {cache: {"cache_storage":...}}
open_files = fsspec.open_files(cached_filenames, **kwargs)
```

Reply:
Fixed in https://github.com/pytroll/trollflow2/pull/205/commits/f2701dbf0119780768b780fa721222b166f14d35